change/@fluentui-ie11-polyfills-2021-02-02-21-43-53-16718-remove-packages-npmrc.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```

Comment:
All of these should be change type `none`, since it doesn't affect the published package.

Reply:
Thanks